Microsoft.Toolkit.Uwp.UI.Controls/TabView/ITabWidthProvider.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Windows.UI.Xaml.Controls;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// Interface for a provider to determine <see cref="TabViewItem"/> widths for the <see cref="TabView"/>.
+    /// </summary>
+    public interface ITabWidthProvider
```

Comment:
This seems like an overkill. Could this be replaced by three dependency properties on the TabView:
* bool IsSelectedTabWidthDifferent
* int MinimumTabWidth
* enum TabWidthBehavior : EqualWidth | DynamicWidth

Reply:
These should go on the TabViewItem itself. No need for the first two properties at all.

Reply:
Curios what are all the scenarios that need to be covered? Still seems weird having to provide an external measure function

Reply:
We discussed this in the meeting Monday.  I explored a custom panel option which looks promising this morning, so I'll either internalize this for now or swap to that.  May push this customization out for the next rev.